app/models/Profile.php

```diff
@@ -118,8 +118,9 @@ public static function isComplete($id)
   public static function getUserProfile($id)
   {
     $fields = array('birthdate', 'phone', 'address_street as address_1', 'address_premise as address_2', 'city', 'state', 'zip as zip_code', 'gender', 'grade', 'school as current_high_school');
-    return $profile = Profile::where('user_id', $id)->select($fields)->first()->toArray();
-
+    if(Profile::where('user_id', '=', $id)->exists())
+      return $profile = Profile::where('user_id', $id)->select($fields)->first()->toArray();
+    return NULL;
```

Comment:
ah yeah I didn't use brackets because in other places doing the same thing it's formatted this way